experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -704,9 +704,11 @@ export interface RequiredField<TTypes extends AllowedTypes> extends TreeField {
  * @alpha
  */
 export interface OptionalField<TTypes extends AllowedTypes> extends TreeField {
-	readonly content?: UnboxNodeUnion<TTypes>;
+	get content(): UnboxNodeUnion<TTypes> | undefined;
+	set content(newContent: FlexibleNodeContent<TTypes> | undefined);
+
 	readonly boxedContent?: TypedNodeUnion<TTypes>;
-	setContent(content: undefined | FlexibleNodeContent<TTypes>): void;
+	// setContent(content: FlexibleNodeContent<TTypes> | undefined): void; // TODO
```

Comment:
The task left removal of the existing `setContent` APIs as a question. What are folks' thoughts?

Reply:
We should remove it, IMO. @noencke

Reply:
Since the TS 5.1 setters allow us to accept all the types set content would have, and for this field kind there are no additional options we want to be able to provide, and the property we are setting can have documentation about doing the set operation to it, in this case it seems like nothing is gained by having the separate method, so I'm on the side of removing it as well, though if we add options (like first vs last write wins) we might want to bring it back in the future, and we should document the merge semantics on the content property .

Reply:
I'll go ahead and remove then. What are the merge semantics currently?

Reply:
Let's go ahead and doc merge policy and whatnot after the code churn has settled (and let's not forget!).